spec/factories/rubric_criteria.rb

```diff
@@ -5,16 +5,12 @@
     max_mark { 4.0 }
     ta_visible { true }
     peer_visible { false }
-    level_0_name { 'Poor' }
-    level_0_description { 'This criterion was not satisifed whatsoever.' }
-    level_1_name { 'Satisfactory' }
-    level_1_description { 'This criterion was satisfied.' }
-    level_2_name { 'Good' }
-    level_2_description { 'This criterion was satisfied well.' }
-    level_3_name { 'Great' }
-    level_3_description { 'This criterion was satisfied very well.' }
-    level_4_name { 'Excellent' }
-    level_4_description { 'This criterion was satisfied exceptionally well.' }
     sequence(:position)
   end
+
+  factory :rubric_criteria_with_levels do
```

Comment:
This isn't bad, but I don't think this really deserves its own factory, since the original rubric criterion factory used to have levels. Why don't you put the `after(:create)` block into the above factory.